ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -107,41 +112,48 @@ public void SetResizeMode(Border view, string resizeMode)
         /// Set the source URI of the image.
         /// </summary>
         /// <param name="view">The image view instance.</param>
-        /// <param name="source">The source URI.</param>
+        /// <param name="sources">The source URI.</param>
         [ReactProp("src")]
-        public void SetSource(Border view, string source)
+        public void SetSource(Border view, JArray sources)
         {
-            var imageBrush = (ImageBrush)view.Background;
+            // There is no image source
+            if (sources.Count == 0)
+            {
+                return;
```

Comment:
Throw an exception? Or we can just do a Debug.Assert if this is guaranteed not to be the case (we often don't do much error checking on native side because of strict contract guarantees from JavaScript).